src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -0,0 +1,645 @@
+    <value>Defines the visibility of the scrollbar.</value>
+  </data>
+  <data name="Profile_SelectionBackgroundColor.Header" xml:space="preserve">
+    <value>Selection background color</value>
+  </data>
+  <data name="Profile_SelectionBackgroundColorToolTip.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Sets the background color of selected text. Overrides the selection background set in the color scheme.</value>
+  </data>
+  <data name="Profile_SnapOnInput.Content" xml:space="preserve">
+    <value>Scroll to input when typing</value>
+  </data>
+  <data name="Profile_SnapOnInput.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the window will scroll to the command input line when typing. Otherwise, the window will not scroll when you start typing.</value>
+  </data>
+  <data name="Profile_StartingDirectory.Header" xml:space="preserve">
+    <value>Starting directory</value>
+  </data>
+  <data name="Profile_StartingDirectory.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>The directory the shell starts in when it is loaded.</value>
+  </data>
+  <data name="Profile_StartingDirectoryBrowse.Content" xml:space="preserve">
+    <value>Browse</value>
+  </data>
+  <data name="Profile_SuppressApplicationTitle.Content" xml:space="preserve">
+    <value>Suppress title changes</value>
+  </data>
+  <data name="Profile_SuppressApplicationTitle.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the tab title overrides the default title of the tab and any title change messages from the application will be suppressed. Otherwise, the tab title behaves as normal.</value>
+  </data>
+  <data name="Profile_TabTitle.Header" xml:space="preserve">
+    <value>Tab title</value>
+  </data>
+  <data name="Profile_TabTitle.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Replaces the name as the title to pass to the shell on startup. Some shells (like bash) may choose to ignore this initial value, while others (CMD, PowerShell) may use this value over the lifetime of the application.</value>
+  </data>
+  <data name="Profile_TopBackgroundImageAlignment.Text" xml:space="preserve">
+    <value>Top</value>
+  </data>
+  <data name="Profile_TopLeftBackgroundImageAlignment.Text" xml:space="preserve">
+    <value>Top left</value>
+  </data>
+  <data name="Profile_TopRightBackgroundImageAlignment.Text" xml:space="preserve">
+    <value>Top right</value>
+  </data>
+  <data name="Profile_UnderscoreCursorShape.Content" xml:space="preserve">
+    <value>Underscore</value>
+  </data>
+  <data name="Profile_UniformBackgroundImageStretchMode.Content" xml:space="preserve">
+    <value>Uniform</value>
+  </data>
+  <data name="Profile_UniformToFillBackgroundImageStretchMode" xml:space="preserve">
+    <value>Uniform to fill</value>
+  </data>

```

Comment:
```suggestion
  <data name="Profile_CursorShapeUnderscore.Content" xml:space="preserve">
    <value>Underscore</value>
  </data>
  <data name="Profile_BackgroundImageStretchModeUniform.Content" xml:space="preserve">
    <value>Uniform</value>
  </data>
  <data name="Profile_BackgroundImageStretchModeUniformToFill" xml:space="preserve">
    <value>Uniform to fill</value>
  </data>
```
Largest to smallest.

Reply:
Also, shouldn't it be `Profile_BackgroundImageStretchModeUniformToFill.Content`? or `.Text`?